var/spack/repos/builtin/packages/podio/package.py

```diff
@@ -47,10 +53,13 @@ class Podio(CMakePackage):
     depends_on('catch2@3.0.1:', type=('test'), when="@0.13:")
 
     conflicts('+sio', when='@:0.12', msg='sio support requires at least podio@0.13')
+    conflicts('cxxstd=20', when='@:0.14.0', msg='c++20 support was added in 0.14.1')
```

Comment:
If the suggestion above works this is not needed anymore:
```suggestion
```

Reply:
In 002dceb0c3 I added a `conditional('20', when='@0.15:')` for `cxxstd` and removed the `conflicts`. That's stricter than necessary, but it's reasonable to add this feature at 0.15.